### DISCUSSION TITLE
New site down?

### DISCUSSION THREAD

**Discussion by narucx** [tteck/Proxmox#3197] [General] [answered]
404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: fra1::pkvbl-1718362464613-69bbb7bf7ec3
![image](https://github.com/tteck/Proxmox/assets/75908696/9da165ff-17db-4202-9bf4-b2d919dee493)

**Comment by tteck** [comment, marked as answer]
Clear browser cache

**Reply by narucx** [reply]
Is it just temporary a redirect to the old site/design or are you going back to that?

**Reply by tteck** [reply]
Back to the old site.

**Reply by asylumexp** [reply]
is there a specific reason? just wondered when i noticed it returned to the old site

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/3081#discussioncomment-9754092
